client/components/domains/domain-search-results/index.jsx

```diff
@@ -36,9 +35,8 @@ var DomainSearchResults = React.createClass( {
 		onAddMapping: React.PropTypes.func,
 		onClickMapping: React.PropTypes.func
 	},
-	isDomainUnavailable: function() {
-		return this.props.lastDomainError &&
-			includes( [ 'not_available', 'not_available_but_mappable' ], this.props.lastDomainError.code );
+	isDomainMappable: function() {
+		return this.props.lastDomainError && 'not_available_but_mappable' === this.props.lastDomainError.code;
```

Comment:
I agree that yoda condition does not improve readability (though, that's usually not its purpose) - it ended up that way because I was modifying existing code and there the string was first and the variable second, so I went with it. I agree that reversing it makes it more readable. On the other hand, the `(this.props.lastDomainError || {}).code === 'not_available_but_mappable` seems way over-engineered to me 😆